stable_baselines3/common/on_policy_algorithm.py

```diff
@@ -69,6 +70,7 @@ def __init__(
         seed: Optional[int] = None,
         device: Union[th.device, str] = "auto",
         _init_setup_model: bool = True,
+        supported_action_spaces: Optional[Tuple[gym.spaces.Space, ...]] = None,
```

Comment:
Why support for no list? I see this could cause nice, silent errors in future. Easier to just require list of supported action spaces always, no?

Reply:
I think because `isinstance()` expect a tuple.
And if you pass a list it would work too but for me, when you use a tuple, it is clearer that is unmutable.

>Easier to just require list of supported action spaces always, no?

The optional is to avoid breaking derived algorithms like in the contrib.